cli/pcluster/configure/easyconfig.py

```diff
@@ -324,6 +325,13 @@ def _prompt_for_subnet(default_subnet, all_subnets, qualified_subnets, message):
     return prompt_iterable(message, qualified_subnets, default_value=default_subnet)
 
 
+def _is_instance_type_supported_for_head_node(instance_type):
+    if instance_type in HEADNODE_UNSUPPORTED_INSTANCE_TYPES:
+        print("The instance type '{0}' is not supported as head node.".format(instance_type))
```

Comment:
we should define a variable like HEAD_NODE_MESSAGES in the validators.py to be used in both the places